### DISCUSSION TITLE
could you add scipy package?

### DISCUSSION THREAD

**Discussion by l1t1** [termux/termux-app#3251] [Q&A] [answered]
https://wiki.termux.com/wiki/Python says
>A few python packages are available from termux's package manager (for python3 only), and should be installed from there to avoid compilation errors. This is the case for:

>numpy, pkg install python-numpy
electrum, pkg install electrum
opencv, pkg install opencv-python
asciinema, pkg install asciinema
matplotlib, pkg install matplotlib
cryptography, pkg install python-cryptography

I try to use `pip install scipy` and encounter build error.

**Comment by 2096779623** [comment, marked as answer]
The `scipy` is available in [termux-user-repository](https://github.com/termux-user-repository/tur) , install it with the following command:
```bash
pkg i tur-repo -y
pkg i python-scipy -y
```

**Comment by l1t1** [comment]
thanks

**Comment by l1t1** [comment]
pkg i python-pandas is also works,thanks again.
